### DISCUSSION TITLE
CTRLR site (forum and downloads) accessible again

### DISCUSSION THREAD

**Discussion by bijlevel** [RomanKubiak/ctrlr#577] [General]
To my surprise I was able to use the former ctrlr.org site today again without a login. I could search the forum section and access the (panel) downloads. If this is permanent that would be very welcome! (Or was this already the case?)

**Comment by damiensellier** [comment]
I think the ctrlr org website problem is being read only. There was no problem downloading the panels from the panels page BUT the problem is coming from the forum. You need to be logged in to be able to download panels and files from there and logging in is impossible.

**Reply by bijlevel** [reply]
Yeah, I realise now it's been this way already for a while. Still a good source of valuable information!
